pkg/templates/installation_context.go

```diff
@@ -42,21 +42,11 @@ func (ctx *InstallationContext) releaseMeta() string {
 	return string(data)
 }
 
-func (ctx *InstallationContext) entitlementsYAML() string {
-	data, err := yaml.Marshal(ctx.Meta.Entitlements)
-	if err != nil {
-		level.Error(ctx.Logger).Log("msg", "unable to marshal entitlements", "err", err)
-		return ""
-	}
-	return string(data)
-}
-
 func (ctx *InstallationContext) FuncMap() template.FuncMap {
 	return template.FuncMap{
-		"ReleaseMeta":       ctx.releaseMeta,
```

Comment:
Was this already in a published release? Will removing this break any unknown usage?

Reply:
@marccampbell this was never released